source/Octopus.Cli/Commands/Releases/PreventReleaseProgressionCommand.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Threading.Tasks;
+using Octopus.Cli.Infrastructure;
+using Octopus.Cli.Repositories;
+using Octopus.Cli.Util;
+using Octopus.Client;
+using Octopus.Client.Exceptions;
+using Octopus.Client.Model;
+
+namespace Octopus.Cli.Commands.Releases
+{
+    [Command("prevent-release-progression", Description = "Prevent a release from progressing to next phase.")]
```

Comment:
I am sure that you have already had this conversation, but this command name is not paired up with the `unblock-release`

Reply:
Sorry had to reopen a new branch to resolve the `CLA` thing, as I'm lazy to change every author of my commit history.

Reply:
At the moment, I'm trying to keep it consistent with UI text, `Prevent Progression` and `Unblock`. I'm sure there may have a discussion in this PR about the naming, and hopefully we will get a better naming convention that make most of us happy.